docs/en/pact-reference.md

```diff
@@ -453,8 +454,44 @@ children:
 }
 ```
 
-#### Pact values returned {#pact-values}
-A successful pact execution will return a value that is valid JSON. A pact
+#### Pact success values {#pact-success}
+
+A successful pact execution returns the output value and any events that may have been emitted
+during the course of the transaction. If there are no events, the JSON contains a single [Pact Value](#pact-values).
+If there are events, the pact value and events are returned in the following JSON structure:
+
+##### `value`
+_type:_ **[PactValue](#pact-values)**
+Pact result value.
+
+##### `events`
+_type:_ **[ [PactEvent](#pact-event) ]**
+Pact events.
+
+#### Pact events {#pact-event}
+
+Pact events represent named, provable events that transpired in a transaction, in the following format:
+
+##### `name`
+_type:_ **string** `required`
+Event name or "topic"
+
+##### `params` `required`
+_type:_ **[ [PactValue](#pact-values) ]**
+Values representing the parameterization of this event.
+
+##### `module`
+_type:_ **string** `required`
+Fully-qualified module name that sourced the event.
+
+##### `moduleHash`
+_type:_ **string (base64url)** `required`
+Hash of sourcing module.
+
+
+#### Pact Values {#pact-values}
```

Comment:
For consistency, `Pact Values` could be `Pact values` since the rest of the section titles use this capitalization scheme.